### PR TITLE
fix indentation issue with autoscaler=hpa

### LINKED ISSUE #68: memory limit not indented properly
There's an indentation bug in `spin kube scaffold` with `--memory-limit` set. This causes some commands like `helmify` to fail.

```
><> spin kube scaffold -f bacongobbler/myapp:1.0.0 --replicas 3 --autoscaler hpa --cpu-limit 100m --memory-limit 128Mi
apiVersion: core.spinoperator.dev/v1alpha1
kind: SpinApp
metadata:
  name: myapp
spec:
  image: "bacongobbler/myapp:1.0.0"
  executor: containerd-shim-spin
  enableAutoscaling: true
  resources:
    limits:
      cpu: 100m
      memory: 128Mi
---
apiVersion: autoscaling/v2
kind: HorizontalPodAutoscaler
metadata:
  name: myapp-autoscaler
spec:
  scaleTargetRef:
    apiVersion: apps/v1
    kind: Deployment
    name: myapp
  minReplicas: 3
  maxReplicas: 3
  metrics:
  - type: Resource
    resource:
      name: cpu
      target:
        type: Utilization
        averageUtilization: 60
  - type: Resource
    resource:
     name: memory
      target:
        type: Utilization
        averageUtilization: 60
```

Expected:

```
  - type: Resource
    resource:
      name: memory
      target:
        type: Utilization
        averageUtilization: 60
```

Error:

```
><> spin kube scaffold -f bacongobbler/myapp:1.0.0 --replicas 3 --autoscaler hpa --cpu-limit 100m --memory-limit 128Mi | ./helmify
INFO[0000] Chart name is not set. Using default name 'chart
ERRO[0000] unable to decode yaml from input              error="error converting YAML to JSON: yaml: line 22: mapping values are not allowed in this context"
```

### PR DESCRIPTION
**Pull request by bacongobbler**
closes #68

### CODE REVIEW

File: pkg/cmd/testdata/keda_autoscaler.yml [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+      memory: 128Mi
+---
+apiVersion: keda.sh/v1alpha1
+kind: ScaledObject
+metadata:
+  name: example-app-autoscaler
+spec:
+  scaleTargetRef:
+	apiVersion: apps/v1
+	kind: Deployment
```
Comment (rajatjindal) [inline pkg/cmd/testdata/keda_autoscaler.yml:21]: these lines look incorrectly indented.

Comment (bacongobbler) [reply pkg/cmd/testdata/keda_autoscaler.yml:21]: Good catch. I’ll go fix that as well.

Comment (bacongobbler) [reply pkg/cmd/testdata/keda_autoscaler.yml:21]: Looks like it's due to the source file using tabs and spaces for indendation. Will fix

Comment (rajatjindal) [reply pkg/cmd/testdata/keda_autoscaler.yml]: Thank you Matt.

**Pull request merged**